numpy/lib/histograms.py

```diff
@@ -779,10 +779,13 @@ def histogramdd(sample, bins=10, range=None, normed=False, weights=None):
 
     Parameters
     ----------
-    sample : array_like
-        The data to be histogrammed. It must be an (N,D) array or data
-        that can be converted to such. The rows of the resulting array
-        are the coordinates of points in a D dimensional polytope.
+    sample : (N, D) array, or (D, N) array_like
+        The data to be histogrammed.
+        
+        If an array, each row is a coordinate in a D-dimensional space.
```

Comment:
This might look better as a list rather than separate paragraphs. Why the transpose for array_like?

Reply:
Transpose was always there, not my choice - just documented it.

It's so the caller can pass `(X, Y, Z)` instead of `np.stack((X, Y, Z), axis=-1)`, I think, which is similar to some matplotlib APIs.

I don't like it, but I think we're stuck with it.